layouts/_default/single.html

```diff
@@ -6,8 +6,9 @@
       {{ if .Params.tags }}
         <div class="blog-tags">
           {{ range .Params.tags }}
-            <a href="{{ $.Site.BaseURL }}/tags/{{ . | urlize }}/">{{ . }}</a
+            <a href="{{ "" | absLangURL }}/tags/{{ . | urlize }}/">{{ . }}</a
             >&nbsp;
+            <a href="{{ "" | absLangURL }}/tags/{{ . | urlize }}/">{{ . }}</a>&nbsp;
```

Comment:
I think this line is the same as above the line (L9), so it doesn't need.

Reply:
You are right. Only different line break. Just remove it ...